lib/driver.rb

```diff
@@ -1,9 +1,11 @@
 require 'csv'
 require_relative 'trip'
+require 'awesome_print'
 
 module RideShare
   class Driver
-    attr_reader :id, :name, :vehicle_id, :status, :trips
+    attr_reader :id, :name, :vehicle_id
+    attr_accessor :status, :trips
```

Comment:
`trips` should never be an accessor.  You can still push new trips into the array with it as a reader.